com.unity.cinemachine/Samples~/2D Samples/2D Fighters.unity

```diff
@@ -0,0 +1,2098 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
This is looking good.  I would maybe tweak it a little - either enlarge the max ortho size, or decrease damping, or both - because too often some of the non-falling players go offscreen